modules/run-consul/run-consul

```diff
@@ -41,11 +41,22 @@ function print_usage {
-  echo -e "  --key-file-path\t\tPath to the certificate key used to verify incoming connections. Optional. Must be specified with --enable-rpc-encryption and --cert-file-path."
-  echo -e "  --environment\t\A single environment variable in the key/value pair form 'KEY=\"val\"' to pass to Consul as environment variable when starting it up. Repeat this option for additional variables. Optional."
+  echo -e "  --cert-file-path\tPath to the certificate file used to verify incoming connections. Optional. Must be specified with --enable-rpc-encryption and --key-file-path."
+  echo -e "  --key-file-path\tPath to the certificate key used to verify incoming connections. Optional. Must be specified with --enable-rpc-encryption and --cert-file-path."
+  echo -e "  --environment\t\tA single environment variable in the key/value pair form 'KEY=\"val\"' to pass to Consul as environment variable when starting it up. Repeat this option for additional variables. Optional."
   echo -e "  --skip-consul-config\tIf this flag is set, don't generate a Consul configuration file. Optional. Default is false."
   echo
+  echo "Options for enabling Consul Autopilot:"
+  echo
+  echo -e "  --cleanup-dead-servers\tSet to true or false to control the automatic removal of dead server nodes periodically and whenever a new server is added to the cluster. Defaults to true. Optional."
+  echo -e "  --last-contact-threshold\tControls the maximum amount of time a server can go without contact from the leader before being considered unhealthy. Must be a duration value such as 10s. Defaults to 200ms. Optional."
+  echo -e "  --max-trailing-logs\t\tControls the maximum number of log entries that a server can trail the leader by before being considered unhealthy. Defaults to 250. Optional."
+  echo -e "  --server-stabilization-time\tControls the minimum amount of time a server must be stable in the 'healthy' state before being added to the cluster. Only takes effect if all servers are running Raft protocol version 3 or higher. Must be a duration value such as 30s. Defaults to 10s. Optional."
+  echo -e "  --redundancy-zone-tag\t\t(Enterprise-only) This controls the -node-meta key to use when Autopilot is separating servers into zones for redundancy. Only one server in each zone can be a voting member at one time. If left blank (the default), this feature will be disabled. Optional."
+  echo -e "  --disable-upgrade-migration\t(Enterprise-only) If this flag is set, this will disable Autopilot's upgrade migration strategy in Consul Enterprise of waiting until enough newer-versioned servers have been added to the cluster before promoting any of them to voters. Optional."
+  echo -e "  --upgrade-version-tag\t\t(Enterprise-only) That tag to be used to override the version information used during a migration. Optional."
```

Comment:
Building on Jim's comment in https://github.com/hashicorp/terraform-aws-consul/pull/82/files#r217219284, once you declare `readonly` variables for these properties, you can use those to populate the default values here programmatically. For example:

```bash
readonly LAST_CONTACT_THRESHOLD_DEFAULT="200"
...
--last-contact-threshold\tControls the ... Value specified in ms. Default: $LAST_CONTACT_THRESHOLD. Optional."
```